app/models/manageiq/providers/redhat/inventory/parser/infra_manager.rb

```diff
@@ -593,4 +598,27 @@ def extract_host_os_name(host)
   def extract_host_os_full_version(host_os)
     host_os.dig(:version, :full_version)
   end
+
+  def get_ip_addresses(addresses, device, nets)
```

Comment:
all the same comments as for the other copy.
Maybe we can use this method in the v4 or the other way around.
@pkliczewski what do you think.